client/src/components/productState.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState, createContext} from 'react';
```

Comment:
nice use of usestate! haven't heard of this before but it looks useful for passing state